### DISCUSSION TITLE
Is it possible to make the text & images editable ?

### DISCUSSION THREAD

**Discussion by xd003** [ocrmypdf/OCRmyPDF#1202] [Q&A] [answered]
Thanks for this wonderful tool. I was just wondering if it can make the text and images editable ( not just searchable ). For example, Adobe Acrobat has these options in their OCR settings 
![IMG_20231201_171035_240](https://github.com/ocrmypdf/OCRmyPDF/assets/26503185/9cc2fb2b-cf69-444d-b76f-6361ae3d96cc)

**Comment by jbarlow83** [comment, marked as answer]
There's an experiment API that allows editing text (see `ocrmypdf.api`) but it will require third party tooling or at least tweaking HTML-like files by hand. gscan2pdf is planning to support text editing this way.
